.gitignore

```
@@ -9,3 +9,4 @@ test/browser/sample/appinfo.js
 dist/aws-sdk-all.js
 yarn.lock
 package-lock.json
+workspace
```

This diff is not required for this code change
```suggestion
workspace
```